Sources/MapboxMaps/Foundation/Camera/CameraView.swift

```diff
@@ -166,83 +42,42 @@ internal class CameraView: UIView {
                              pitch: localPitch)
     }
 
-    private unowned var delegate: CameraViewDelegate!
-
-    init(delegate: CameraViewDelegate, edgeInsets: UIEdgeInsets = .zero) {
-        self.delegate = delegate
+    init() {
```

Comment:
is this required to have, or is it implied?

Reply:
It's required.. we're setting some properties in the `init`.